bin/bundle

```diff
@@ -39,8 +39,11 @@ m = Module.new do
 
   def gemfile
     gemfile = ENV["BUNDLE_GEMFILE"]
+    # rubocop:disable Rails/Present
     return gemfile if gemfile && !gemfile.empty?
```

Comment:
I'm not able to realize why is this line different from `gemfile.present?`. 🤔 

On the other hand, here you can use an inline comment to disable the rubocop rule only there, like this:
```ruby
return gemfile if gemfile && !gemfile.empty? # rubocop:disable Rails/Present
```

Reply:
At some point, when executing rubocop it gave me an error about putting gemfile.present. Now I tried and works fine 👍